src/components/NewRuntimeModal.js

```diff
@@ -768,6 +772,7 @@ export const NewRuntimeModal = withModalDrawer({ width: 675 })(class NewRuntimeM
               id => h(Fragment, [
                 label({ htmlFor: id, style: styles.label }, ['Workers']),
                 h(NumberInput, {
+                  'aria-label': 'Number of workers in cloud environment',
```

Comment:
So actually, this shouldn't be necessary since the input is already bound to a label using `htmlFor`.

I did have to do a fix in my PR to the NumberInput label assertion, though, to make sure it actually pays attention to the `id` field if there is one. Merge that branch into this one and you'll see these assertions should go away